builder/assignmentEnvImageBuilder.go

```diff
@@ -88,7 +100,10 @@ func (assgnEnv *assignmentEnvironment) verifyAndWriteInstructions() error {
 	return nil
 }
 
-func (assgnEnv *assignmentEnvironment) verifyLanguage() error {
+// verifyLanguage searches the docker image for the given language of docker hub.
+// It uses ImageSearch function of docker client (here - https://github.com/moby/moby/tree/master/client)
+// and returns error if image is not found.
```

Comment:
no need to mention that an error is returned.

Reply:
Done